docs/lts.md

```diff
@@ -132,7 +132,7 @@ If there are other ways to set this up on MacOS please considering sending a pul
 
 - No homebrew, they don't have arm builds, we will include [Pixi](https://github.com/prefix-dev/pixi) instead, which is a nice package manager built on the popular conda ecosystem - there's some good stuff in there!
 - Chromium 
-- No Firefox or Thunderbird, missing ARM builds, just about everything else is there.
+- No Firefox missing ARM builds, just about everything else is there.
```

Comment:
How did you install Thunderbird?  I couldn't see ARM in the Flatpak metadata

Reply:
in the flatpak list in /etc/ublue-whatever, I removed "Firefox" yesterday on my first install, because it was still there. Then I ran the install-system-flatpaks action again and had Thunderbird in my dock.